csrc/loader.cpp

```diff
@@ -32,9 +32,6 @@ void initialize()
     CRYPTO_library_init();
     ERR_load_crypto_strings();
     OpenSSL_add_all_digests();
-
-    // seed the PRNG
-    RAND_poll();
```

Comment:
we don't need to do this anymore?

Reply:
ah, i see. it's a no-op in AWS-LC

https://github.com/aws/aws-lc/blob/4368aaa6975ba41bd76d3bb12fac54c4680247fb/crypto/rand_extra/rand_extra.c#L49

Reply:
Yes. [This is no op](https://github.com/aws/aws-lc/blob/AWS-LC-FIPS-2.0.13/include/openssl/rand.h#L90) and it's marked deprecated in v1.30.1.